cmd/kube-apiserver/app/aggregator.go

```diff
@@ -188,6 +188,7 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	{Group: "rbac.authorization.k8s.io", Version: "v1beta1"}:     {group: 17000, version: 12},
 	{Group: "rbac.authorization.k8s.io", Version: "v1alpha1"}:    {group: 17000, version: 9},
 	{Group: "settings.k8s.io", Version: "v1alpha1"}:              {group: 16900, version: 9},
+	{Group: "settings.k8s.io", Version: "v1beta1"}:               {group: 16900, version: 9},
```

Comment:
1. we should find out what the behavior of matching priorities for different versions of the same group is
2. how about make v1beta1 lower priority for now?